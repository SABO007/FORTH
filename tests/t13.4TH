: SQUARE DUP * ;
: CUBE DUP SQUARE * ;
: QUAD SQUARE SQUARE ;

5 SQUARE .
3 CUBE .
2 QUAD .