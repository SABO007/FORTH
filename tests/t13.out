25
27
16
